src/server/game.js

```diff
@@ -6,10 +6,19 @@ const db = require("./database/manager");
 const msgPack = require("msgpack-lite");
 const gameConstants = require("../game-constants");
 
-function Marble(id, entryId, name, color) {
+const anyColorAllowed = ["abstract", "marble", "default", "swirly", "squares"]; // Will (probably) be changed to a meta file later on, next to the skin file?
+
+function Marble(id, entryId, name, options = {}) {
```

Comment:
Just a tiny thing, but since it's `attributes` everywhere else now, you might want to rename it here too.